packages/shared-ux/page/solution_nav/src/with_solution_nav.tsx

```diff
@@ -71,9 +69,14 @@ export const withSolutionNav = <P extends ComponentProps>(WrappedComponent: Comp
       />
     );
 
-    const pageSideBarProps = {
+    const pageSideBarProps: TemplateProps['pageSideBarProps'] = {
       paddingSize: 'none' as 'none',
       ...props.pageSideBarProps,
+      // TODO: `minWidth` isn't re-populating down on state change
+      minWidth:
+        isMediumBreakpoint || (canBeCollapsed && isLargerBreakpoint && !isSideNavOpenOnDesktop)
+          ? euiTheme.size.xxl
+          : undefined,
```

Comment:
Help needed here: I didn't have time to track down why this isn't updating. Maybe it needs a `useMemo` or something?

Reply:
Fixed in https://github.com/elastic/eui/pull/6191